backend/core/src/migration/1675268723433-migration-reset.ts

```diff
@@ -0,0 +1,717 @@
+        [jurisdictionName, jurisDefault, "Example Terms Go here"]
+      )
+    }
+    for (const unitType of [
+      { num_bedrooms: 0, name: "SRO" },
+      { num_bedrooms: 0, name: "studio" },
+      { num_bedrooms: 1, name: "oneBdrm" },
+      { num_bedrooms: 2, name: "twoBdrm" },
+      { num_bedrooms: 3, name: "threeBdrm" },
+      { num_bedrooms: 4, name: "fourBdrm" },
+      { num_bedrooms: 5, name: "fiveBdrm" },
+    ]) {
+      await queryRunner.query(`INSERT INTO "unit_types" (name, num_bedrooms) VALUES ($1, $2)`, [
+        unitType.name,
+        unitType.num_bedrooms,
+      ])
+    }
+    for (const priorityType of [
+      "Mobility",
+      "Hearing",
+      "Visual",
+      "Hearing and Visual",
+      "Mobility and Hearing",
+      "Mobility and Visual",
+      "Mobility, Hearing and Visual",
+    ]) {
+      await queryRunner.query(
+        `INSERT INTO "unit_accessibility_priority_types" (name) VALUES ($1)`,
+        [priorityType]
+      )
+    }
+    for (const unitRentType of ["fixed", "percentageOfIncome"]) {
+      await queryRunner.query(`INSERT INTO "unit_rent_types" (name) VALUES ($1)`, [unitRentType])
+    }
+
+    const [{ id: jurisdictionId }] = await queryRunner.query(
+      `SELECT id FROM jurisdictions WHERE name = 'Alameda' LIMIT 1`
+    )
+
+    for (const communityType of ["specialNeeds", "senior55", "senior62"]) {
+      await queryRunner.query(
+        `INSERT INTO reserved_community_types (name, jurisdiction_id) VALUES ($1, $2)`,
+        [communityType, jurisdictionId]
+      )
+    }
+
+    const translations = {
+      changeEmail: {
+        message: "An email address change has been requested for your account.",
+        onChangeEmailMessage:
+          "To confirm the change to your email address, please click the link below:",
+        changeMyEmail: "Confirm email change",
+      },
+      invite: {
+        hello: "Welcome to the Partners Portal",
+        inviteMessage:
+          "Welcome to the Partners Portal on %{appUrl}. You will now be able to manage listings and applications that you are a part of from one centralized location.",
+        inviteManageListings:
+          "You will now be able to manage listings and applications that you are a part of from one centralized location.",
+        inviteWelcomeMessage: "Welcome to the Partners Portal at %{appUrl}.",
+        toCompleteAccountCreation:
+          "To complete your account creation, please click the link below:",
+        confirmMyAccount: "Confirm my account",
+      },
+      mfaCodeEmail: {
+        message: "Access token for your account has been requested.",
+        mfaCode: "Your access token is: %{mfaCode}",
+      },
+      header: {
+        logoUrl:
+          "https://res.cloudinary.com/exygy/image/upload/v1652459319/housingbayarea/163838489-d5a1bc08-7d69-4c4a-8a94-8485617d8b46_dkkqvw.png",
+        logoTitle: "Alameda County Housing Portal",
```

Comment:
Maybe this was existing but I didn't think we had references to Alameda here, dev is just "Housing Portal"

Reply:
so I pulled these translations directly from the old migrations, I think we probably updated some of the translations directly through queries instead of through migrations 

thats a good thing to flag and another reason we probably don't want to reseed staging/production envs

nice catch!